server/statistics/statistics.go

```diff
@@ -87,7 +80,8 @@ func NewStatistics(logger logger.Logger) *Statistics {
 		broadcasts_total: NewStatsValue(),
 	}
 	go result.PrintStats()
-	//go result.TestStats()
+	//Enable the following line for testing statistics gathering and aggregation
+	go result.TestStats()
```

Comment:
Do you want to keep it enabled, or is it better to have it commented out?

Reply:
Damn didnt see this before merge...